### PR TITLE
FFWEB-1278 Added FAQ section to the documentation.

### PR DESCRIPTION
**Pull request by wdutkiewicz-obj**

### CODE REVIEW

File: markdown/3.x/en/faq/errors.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## FAQ - Errors
+
+---
+#### Error: 
+
+could not invoke subscriber function [undefined] for topic [records]. error: TypeError: Cannot read property 'insertAll' of undefined Line 2 Bundle.js
+
+#### Solution:
+
+Add missing `defer` attribute on bundle.js like described [here](https://web-components.fact-finder.de/documentation/3.x/include-scripts): `<script defer src="../dist/bundle.js"></script>`
```
Comment (torsten-knauf) [inline markdown/3.x/en/faq/errors.md]:
```suggestion
Add missing `defer` attribute on bundle.js like described [here](/documentation/3.x/include-scripts): `<script defer src="../dist/bundle.js"></script>`
```

File: src/views/faq-view.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
```
[the remark this answers is not in the document]
Comment (wdutkiewicz-obj) [reply src/views/faq-view.js]: I don't know. I just made a copy of existing component and adjusted it. I think it may stay.

**Comment by wdutkiewicz-obj** [comment]
> 
> 
> I feel the subpages are too much overhead (yet) and rather make it less overseeable. I would prefer something like a list of content at the beginning and then everything on one page. The list of content can be structured in subsections like _General_ and _Errors_.
> 
> If it becomes messy after a time, we can re-evaluate / re-order and maybe add subpages again.
> 
> Probably adding our search field and _contact us if you haven't found what you were looking for_ at the very top is nice.
> 
> For inspiration I overflew [best examples FAQ](https://www.searchenginejournal.com/best-faq-page-examples/267709/#close)
> 
> Just my first suggestion; I am curious about the ideas of the others.

I don't know how it should look like - according to the ticked we need some FAQ section, no specific requirements. Initially I wanted to put everything on a single page, but in the end I decided to add some structure that somehow fits into existing one. @toarm: what do you think about it? Should I change it or maybe let's leave it and we can rework it in the future. I think the main goal is achieved - we have the FAQ section with some initial content.

**Comment by toarm** [comment]
> > I feel the subpages are too much overhead (yet) and rather make it less overseeable. I would prefer something like a list of content at the beginning and then everything on one page. The list of content can be structured in subsections like _General_ and _Errors_.
> > If it becomes messy after a time, we can re-evaluate / re-order and maybe add subpages again.
> > Probably adding our search field and _contact us if you haven't found what you were looking for_ at the very top is nice.
> > For inspiration I overflew [best examples FAQ](https://www.searchenginejournal.com/best-faq-page-examples/267709/#close)
> > Just my first suggestion; I am curious about the ideas of the others.
> 
> I don't know how it should look like - according to the ticked we need some FAQ section, no specific requirements. Initially I wanted to put everything on a single page, but in the end I decided to add some structure that somehow fits into existing one. @toarm: what do you think about it? Should I change it or maybe let's leave it and we can rework it in the future. I think the main goal is achieved - we have the FAQ section with some initial content.

I think everything should be on one page. When I use an FAQ section I am used to `ctrl + f` shortcut to find a question/answer to my current issue.

I am fine with implementin a sidebar which scrolls to a specific faq entry. That way we could still distinguish between erros and other things but keep everything on on page. 

Like:
General
- E1
- E2
-E3
Errors
- E4
- E5
...

But the content page is stays always the same 
E1
E2
E3
E4
E5
...

**Comment by toarm** [comment]
I think as soon as we exceed 20-30 entries we can think of something more sophisticated.

**Comment by toarm** [comment]
for the angular data-binding issue the link to our github discussion is missing

**Comment by torsten-knauf** [comment]
Please add an entry for deprecated Web APIs as well (see internal chat [here](https://chat.omikron.net:8066/omikron/pl/rig16wgsk7ymte6y3o45fh373h) ff)

**Comment by wdutkiewicz-obj** [comment]
I tried to rework the FAQ so that we have categories in the left and a single page with the content. However, I was unable to implement scrolling, possibly because of Shadow DOM used. My questions are:
1. Can I implement FAQ section without side menu (a single page, something like 'download' page).
2. Do we need separate FAQ for both versions (1.x and 3.x)? It would be easier to put all entries on a single page when there's no switch between 1.x and 3.x.

I don't want to put much effort in this feature as we have 2 (maybe 3, when including the one from Torsten's comment) entries. Right now we don't need a sophisticated page - what I added was just a proposal to start the discussion.

**Comment by toarm** [comment]
> Can I implement FAQ section without side menu (a single page, something like 'download' page).
Yes please do so. Skip the entire left menu.

> Do we need separate FAQ for both versions (1.x and 3.x)? It would be easier to put all entries on a single page when there's no switch between 1.x and 3.x.
Yes, we will have to maintain to distinct versions, even if they overlap at some points.

> I don't want to put much effort in this feature as we have 2
It's getting more and more. Lets merge this fast and see

**Comment by wdutkiewicz-obj** [comment]
Can you have a look now?

**Comment by torsten-knauf** [comment]
For a fast merge it is ok to me. 

> Please add an entry for deprecated Web APIs as well (see internal chat [here](https://chat.omikron.net:8066/omikron/pl/rig16wgsk7ymte6y3o45fh373h) ff)

Should I put this one into a new Jira issue for us?

**Comment by wdutkiewicz-obj** [comment]
> 
> 
> For a fast merge it is ok to me.
> 
> > Please add an entry for deprecated Web APIs as well (see internal chat [here](https://chat.omikron.net:8066/omikron/pl/rig16wgsk7ymte6y3o45fh373h) ff)
> 
> Should I put this one into a new Jira issue for us?

Yes, please add a new ticket. Do you have any idea what I should add there? I'm not sure if I understand the concerns mentioned by Holly.

**Pull request merged**